packages/yoastseo/src/languageProcessing/helpers/sanitize/sanitizeString.js

```diff
@@ -1,16 +1,17 @@
-/** @module stringProcessing/sanitizeString */
-
+import excludeTableOfContentsTag from "./excludeTableOfContentsTag";
 import { stripFullTags as stripTags } from "./stripHTMLTags.js";
 
 import stripSpaces from "./stripSpaces.js";
 
 /**
- * Strip HTMLtags characters from string that break regex
+ * Sanitizes the text before we use the text for the analysis.
+ *
+ * @param {String} text The text to be sanitized.
  *
- * @param {String} text The text to strip the characters from.
- * @returns {String} The text without characters.
+ * @returns {String} The sanitized text.
  */
 export default function( text ) {
+	text = excludeTableOfContentsTag( text );
 	text = stripTags( text );
 	text = stripSpaces( text );
```

Comment:
`stripTags` already calls `stripSpaces`, so this line can be removed.

Reply:
Good point! I'll remove it!